akka-persistence-tck/src/main/scala/akka/persistence/CapabilityFlags.scala

```diff
@@ -52,3 +52,9 @@ trait SnapshotStoreCapabilityFlags extends CapabilityFlags {
   // no flags currently
 }
 //#snapshot-store-flags
+
+trait CurrentEventsByTagCapabilityFlags extends CapabilityFlags {
```

Comment:
@patriknw The JDBC plugin uses a field called `created` for the journal and snapshot table, which is the unix timestamp (System.currentTimeMillis). (Fingers crossed) which should in the same stream between different queries thus the offset can be used as an unique position in the event stream. I understand that this is a bold statement and depends on the database implementation in use.